src/main/java/net/pms/network/mediaserver/nettyserver/RequestV2.java

```diff
@@ -287,11 +288,14 @@ public ChannelFuture answer(
 		}
 
 		if (uri.startsWith("api/")) {
-			ApiHandler api = new ApiHandler();
-			api.handleApiRequest(method, content, output, uri.substring(4), event);
-			ChannelFuture future = event.getChannel().write(output);
-			if (close) {
-				future.addListener(ChannelFutureListener.CLOSE);
```

Comment:
Is it intentional that this close handler was removed? Just checking

Reply:
yes, the API call doesn't return right away, but later (and then it will be closed).